azimuth_capi/status.py

```diff
@@ -142,6 +143,23 @@ def _reconcile_cluster_phase(cluster):
         cluster.status.phase = ClusterPhase.UNHEALTHY
     else:
         cluster.status.phase = ClusterPhase.READY
+        # reset the timeout timestamp, as we are now in a stable state
+        cluster.status.last_updated_timestamp = None
```

Comment:
This is not required given the next statement

Reply:
This is required, I think, I have added more comments.

Reply:
oops, I see what you mean!